.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: generic
-sudo: false
+sudo: true
```

Comment:
:memo: I think you can leave this set to `false`. See #28 for example.

Reply:
Thanks! I'll try that

Reply:
Libjson-rpc-cpp required a slight modification to run on ubuntu 14 (https://gist.github.com/stefanofiorentino/594858159c23841ec4eec56184852d51) so I pointed it to my own fork, and am building it from source. Is this sustainable or should we look for some other solution